py/desispec/io/__init__.py

```diff
@@ -10,7 +10,8 @@
 # The line above will help with 2to3 support.
 
 import warnings
-warnings.filterwarnings('ignore', message="'.*nanomaggies.* did not parse as fits unit.*")
+warnings.filterwarnings('ignore', message="'.*nanomaggie.* did not parse as fits unit.*")
+warnings.filterwarnings('ignore', message="'.*nanomaggy.* did not parse as fits unit.*")
 warnings.filterwarnings('ignore', message=".*'10\*\*6 arcsec.* did not parse as fits unit.*")
```

Comment:
This isn't a change that you made, but I'm puzzled by this: where does `10**6 arcsec` come from?

Reply:
This was added as part of PR #717 to update to a new fibermap format, but I don't know what had/has that oddly scaled unit.

Reply:
OK, noted. Fine to leave it for the purposes of this PR.